src/style.css

```diff
@@ -31,7 +40,18 @@ header {
 }
 
 .btn {
-  height: 30px;
+  height: 50px;
+  font-size: 25px;
+  padding: 5px;
+}
+
+.score {
+  padding: 10px;
+  width: 200px;
+}
+
+.score:nth-child(odd) {
+  background-color: rgba(128, 128, 128, 0.39);
 }
 
 .form {
```

Comment:
- [ ] **Styling:**
- I would advise you to properly style your App and its layout should look like the wireframe below. This is part of the project requirement.
![Capture](https://user-images.githubusercontent.com/63972670/225361496-89ed3bce-3a35-4cd9-97b4-a5fa6a5a9b75.PNG)

Reply:
As the previous reviewer requested, please follow the wireframe. Currently, it looks like below.
<img width="1440" alt="Screenshot 2023-03-15 at 9 41 42 PM" src="https://user-images.githubusercontent.com/25887253/225412014-e56022b0-e07e-486b-b5f5-c77abd48fd8a.png">